### ISSUE TITLE
[BUG]: redundant variables

### ISSUE THREAD

**Issue by fadnincx** [projectpotos/ansible-role-potos_firstboot#14]
### Description of the bug

the variables `potos_firstboot_local_user_add_with_sudo` and `potos_firstboot_local_user_add_without_sudo` are redundent by that generally it's expected that
 ```
potos_firstboot_local_user_add_with_sudo !=  potos_firstboot_local_user_add_without_sudo
```
This should be combined

### Steps To Reproduce

-

**Comment by nis65** [comment]
Hi @fadnincx : just stumbled upon this one. I guess this is fixed by #17 too, as this removed the redundant variables you are complaining in the bug report:

See the [first diff](https://github.com/projectpotos/ansible-role-potos_firstboot/pull/17/files) here.

**Comment by fadnincx** [comment]
Ah, yes. I close this here.
